### ISSUE TITLE
Tooltip text missing on like button on package page

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7315]
Aria-label should be changed to to a 'title' such that the text shows up on tooltip

**Comment by isoos** [comment]
Should we still emit `aria-label` attribute or just `title`?

**Comment by sigurdm** [comment]
As far as I understand when the text is the same we only need the title. I think there is a precedence order for calculating accessible names, and aria-label is considered before title, but when they are the same we don't need both. 

https://www.deque.com/blog/text-links-practices-screen-readers/#:~:text=In%20situations%20where%20text%20that,the%20title%20is%20more%20suitable.

See also https://www.freecodecamp.org/news/web-accessibility-common-aria-mistakes-to-avoid/#:~:text=Don%27t%20Use%20Unnecessary%20ARIA%20Labels
